src/h2load.cc

```diff
@@ -366,9 +438,16 @@ Client::~Client() {
     worker->sample_client_stat(&cstat);
   }
   ++worker->client_smp.n;
+  
+  if (!measurement_calculation_done) {
+    std::cout << "Duration timeout not called on Client: " << id << std::endl;
```

Comment:
Not really a bug/error/warning. This is really for debugging purpose now. So, for some clients, the duration CB is not fired, I wanted to track those. Here we call the duration CB explicitly. I'll remove the printing eventually.